### DISCUSSION TITLE
Changing keybindings

### DISCUSSION THREAD

**Discussion by ajay6915** [pop-os/shell#1090] [Q&A]
Is there a chance to change the keybindings if you had to install the shell on Gnome 3.38 on Debian Bullseye? I have a German keyboard and starting the launcher via Super + / is very inconvenient.

**Comment by ajay6915** [comment]
I just found the shortcuts-application. But I still can't assign different shortcuts.

**Comment by jacobgkau** [comment]
Have you tried the keyboard shortcut settings under Settings -> Keyboard?

**Reply by ajay6915** [reply]
Yes, I did. The keybindings for the Pop-Shell don't appear.

**Reply by AnishDe12020** [reply]
Most if not all of them do exist. Like for the launcher, it is known as "Search". Maybe you can try that

**Reply by ajay6915** [reply]
Yes, I know but on a German keyboard they are partially very inconvenient. For example the app launcher starts with Super + / and the slash is on Shift + 7, so I have to press 3 buttons.

**Reply by pinsard** [reply]
Hi, a bit late, maybe, but I'd like to contribute with a possibility: the keybindings directory is not being copied to /usr/share/glib-2.0/schemas, probably.

I found that out after I installed pop-shell (version 2) on my Debian 12, and didn't find the "Tiling" entry in the Settings, Keyboard, Keyboard Shortcuts menu. Investigating the source code, I could confirm that the files "謹 10-pop-shell-move.xml 謹 10-pop-shell-navigate.xml 謹 10-pop-shell-tile.xml" were not present where they should be. That is why Aja doesn't see them.

```
sudo cp schemas/org.gnome.shell.extensions.pop-shell.gschema.xml /usr/share/glb-2.0/schemas
sudo cp keybindings/*.xml /usr/share/gnome-control-center/keybindings
cd /usr/share/glib-2.0/schemas
sudo glib-compile-schemas .

```
should do it.

**Reply by Okanda** [reply]
You made my day, thank you :)

There's a little error : 
sudo cp schemas/org.gnome.shell.extensions.pop-shell.gschema.xml /usr/share/**glib**-2.0/schemas

Miss the "i"
